demo/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '2'
+services:
+  postgres:
+    image: postgres:9.3
+    container_name: ansible-postgres
+
+  conjur:
+    image: registry.tld/possum:0.1.0-stable
```

Comment:
Now that we're live on the dockerhub & quay.  We can use those instead of our internal repository:
* `cyberark/conjur:0.1.0-stable`
* `quay.io/cyberark/conjur:0.1.0-stable`